specification/keyvault/data-plane/Microsoft.KeyVault/preview/7.2/keys.json

```diff
@@ -975,6 +1028,120 @@
     }
   },
   "definitions": {
+    "KeyReleaseCondition": {
+      "properties": {
+        "claim": {
+          "x-ms-client-name": "claim name",
```

Comment:
This is meant to be a programmatic identifier. You can't use spaces.

Reply:
Fixed